docs/02-guides/magic-button/contribution-mode.mdx

```diff
@@ -26,7 +26,11 @@ The **Magic Button** is only visible when the **Contribution Mode** is active.
 
 The default activation for **Contribution Mode** is defined by the
 [`FRONT_COMMERCE_CONTRIBUTION_MODE_FORCE`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/bbaa7188cdd69e3fdca39859fc696d65019707b4/packages/core/contribution-mode/ContributionMode.ts#L23)
-environment variable
+environment variable.\
+This variable is processed by the
+[`app/config/contributionMode.ts` configuration file](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/update-contribution-mode-demos/skeleton/app/config/contributionMode.ts)
+and used
+[in the Front-Commerce configuration](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/update-contribution-mode-demos/skeleton/front-commerce.config.ts#L19).
 
```

Comment:
**nitpick:** `app/config/contributionMode.ts` only is a convention. It's still unclear IMO.

**suggestion:** state more explicitely what the source of truth is, and how it's in the skeleton

E.g: 
> One can force the contribution mode in `front-commerce.config.ts::contributionMode.force` (link to reference?).
> In a default Front-Commerce application, this is achieved using the `FRONT_COMMERCE_CONTRIBUTION_MODE_FORCE` used in the `app/config/contributionMode.ts`  configuration file. But you can implement your own logic.

Reply:
Very true, I'll update.

Reply:
I also added the missing entry for `contributionMode` in the config API reference.